pkg/apis/core/v1/zz_generated.validations.go

```diff
@@ -8323,7 +8323,6 @@ func Validate_Volume(opCtx operation.Context, obj, oldObj *corev1.Volume, fldPat
 				errs = append(errs, e...)
 				return // do not proceed
 			}
-			errs = append(errs, validate.DNSLabel(opCtx, fldPath, obj, oldObj)...)
```

Comment:
FTR: this is dropped because it says `// +k8s:validation:format="dns-label"` and `FormatValidationFunction()` doesn't handle quoted values.  It could call `strconv.Dequote()` but my PR will make all of these go away anyway.